runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -102,6 +102,7 @@
 {
 	private static final int REFRESH_THROTTLE = 60_000; // ms
 	private static final int TICK_THROTTLE = (int) Duration.ofMinutes(10).toMillis();
+	private static final int MAX_PLAYER_COUNT = 1950;
```

Comment:
I would rather this name not be inaccurate. Instead, define this as `2000` since that is the real max value, and define another constant, `MAX_HOP_TO_PERCENTAGE = 0.975` and use that as the basis for comparison.

Reply:
Amended to match your preferences.